src/browser/modules/Frame/FrameTitlebar.jsx

```diff
@@ -215,6 +223,24 @@ class FrameTitlebar extends Component {
               </DropdownList>
             </DropdownButton>
           </Render>
+          <FrameButton
+            title="Save as Favorite"
+            onClick={() => {
+              props.newFavorite(frame.cmd)
+            }}
+          >
+            <SaveFavorite width={12} />
+          </FrameButton>
+          <Render if={props.isRelateAvailable}>
+            <FrameButton
+              title="Save as project file"
+              onClick={() => {
+                props.newProjectFile(frame.cmd)
```

Comment:
if a file is already saved as a project file , will clicking this then allow users to save again as a completely new file?

Reply:
Yes